### ISSUE TITLE
Active keys saying Unblocked

### ISSUE THREAD

**Issue by bebe123456789** [BOLL7708/OpenVR2Key#96]
### Current behavior

When trying to bind keys or even just use them the active keys say "unblocked". 

### Steps to reproduce the behavior

run game

### Expected behavior

for the software to show the keys/buttons i press

### Operating system version

Windows 11

### SteamVR version

13024612

### Application version

Windows 11

### VR HMD

Oculus Quest 2

### VR hardware

Oculus Quest 2

### Software to control

Steam VR

### Game

Gorilla tag

### Additional information

 I am running steam vr and running a game. Ive tried to look at other people with the same issue as me on the github, since you get asked this a lot., But it didn't really go into depth.

**Comment by BOLL7708** [comment]
Yeah most people solve this on their own and I never get to know what the issue was or how they fixed it, or, I've missed it or forgotten by now. 

Regardless, which bindings are used for the application in the SteamVR controller bindings interface? You might have to toggle on advanced settings and use the old interface if you haven't added this application to your Steam VR Library.

**Comment by timmytimmytim** [comment]
The same thing is happening to me it says unblocked and none of the buttons register when i click them on my controller?

**Comment by BOLL7708** [comment]
> The same thing is happening to me it says unblocked and none of the buttons register when i click them on my controller?

Can you post a screenshot of the application and the SteamVR bindings interface for the application?

**Comment by Reacher123234234** [comment]
I got the exact god damn thing man, it keeps saying unblocked but i have steam vr and gorilla tag running. Should i post a pic?

**Comment by timmytimmytim** [comment]
ill post a pic as soon as I get on my headset

**Comment by timmytimmytim** [comment]
![OpenVR2Key 1_28_2024 3_05_36 PM](https://github.com/BOLL7708/OpenVR2Key/assets/157938087/88aecaeb-6dea-4de5-81d2-749524bf4909)

**Comment by BOLL7708** [comment]
Then you need to check what it says in the SteamVR bindings interface for this application, it depends on the right bindings being loaded to detect input.

**Comment by timmytimmytim** [comment]
how do i do that Im on steamvr meta quest 2

**Comment by BOLL7708** [comment]
`SteamVR > Settings > Controllers > Manage Controller Bindings`

If you haven't added the application to your VR library, you can instead find it by doing the below.

`SteamVR > Settings > Bottom left corner, switch Advanced Settings to Show`
`SteamVR > Settings > Controllers > Show Binding UI`

Then locate OpenVR2Key in there and check which bindings it's using.

**Comment by LajamVR** [comment]
> `SteamVR > Settings > Controllers > Manage Controller Bindings`
> 
> If you haven't added the application to your VR library, you can instead find it by doing the below.
> 
> `SteamVR > Settings > Bottom left corner, switch Advanced Settings to Show` `SteamVR > Settings > Controllers > Show Binding UI`
> 
> Then locate OpenVR2Key in there and check which bindings it's using.

**Comment by LajamVR** [comment]
@BOLL7708 What do i do after i checked which bindings im using

**Comment by BOLL7708** [comment]
Post a screenshot of what you're seeing.

**Comment by ZMoneyRed2** [comment]
![Screenshot 2024-01-17 221628](https://github.com/BOLL7708/OpenVR2Key/assets/158987131/88e7ddbc-d67a-4d39-8c89-23d0c0b73e72)

**Comment by ZMoneyRed2** [comment]
![Controller Binding 2_4_2024 10_25_41 PM](https://github.com/BOLL7708/OpenVR2Key/assets/158987131/5952bffe-a5c2-4164-ad7b-0197e104674e)

**Comment by BOLL7708** [comment]
@ZMoneyRed2 This application supports the newest input system so it shouldn't even get legacy bindings. Did you extract all the files to a folder, including all the bindings files? There should be a `bindings_oculus_touch.json` next to the executable.

**Comment by SLAY3R990** [comment]
you probably have the **latest pre-release** version downloaded instead of the **latest secure** version. _**go to 0.57 and download that instead.**_ _also, make sure to delete your other version. sorry for late reply btw_
